src/kaltura-player.ts

```diff
@@ -830,6 +831,15 @@ export class KalturaPlayer extends FakeEventTarget {
     return this._localPlayer.Error;
   }
 
+  private _addAudioPreset(options: Partial<KalturaPlayerConfig>): void {
+    if (options.plugins?.audioPlayer) {
```

Comment:
this just means that audio player is registered, not that it's loaded.
the plugin is only loaded after it's configuration is parsed successfully. 
are you sure that you want to run this code at this point already ?

also, just as a protection for the future, a better structure would be something like
window.kalturaCustomPresetMap[player id][preset name] = [preset function]
which allows us to support multiple custom presets, if we ever need them. otherwise, if we have only one kalturaCustomPreset variable, and we ever need to add another one, or have the player select between them, it will require a refactor.
but let me know how complicated it would be to make this change now, since it's not directly related to this PR, it's just future proofing

Reply:
I thought it's good enough here cause we just need to make sure audio is exist, actually until now we add it in the beginning , now we add it in case it's audio player and not a video. i didn't think we should handle case it's not loaded because there is other issue but if you think we should change it, where i can put it, what does mean: "configuration is parsed"?
for you second comment - done, i changed it.